Makefile

```diff
@@ -12,7 +12,12 @@ CGO_ENABLED ?= 0
 GOARCH ?= amd64
 
 define compile_service
-	CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM) go build -mod=vendor -ldflags "-s -w" -o ${BUILD_DIR}/mainflux-$(1) cmd/$(1)/main.go
+	CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM) \
+	go build -mod=vendor -ldflags "-s -w \
+	-X 'github.com/mainflux/mainflux.BuildTime=$(TIME)' \
+	-X 'github.com/mainflux/mainflux.Version=$(VERSION)' \
+	-X 'github.com/mainflux/mainflux.Commit=$(COMMIT)'" \
```

Comment:
By default, these (`TIME`, `VERSION`, `COMMIT`) will be empty. Please set them like in `make_dockers`.

Reply:
I we do so we have to install git in the Dockerfile

Reply:
Also, maybe it's better to have for TIME something like `date --rfc-3339=seconds` rather than a custom format.

Reply:
@dusanb94 Let me try what you proposed. It's probably the way to go.
@drasko It is filled in the docker build. Here:
```
		--build-arg VERSION=$(VERSION) \
		--build-arg COMMIT=$(COMMIT) \
		--build-arg TIME=$(TIME) \
```
and here:
```
ARG VERSION
ARG COMMIT
ARG TIME
```